Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2570,6 +2568,195 @@ def pin_message():
         return_error(f"{slack_error}")
 
 
+def list_channels():
+    """
+    List the conversations in the workspace
+    """
+    args = demisto.args()
+    # Default for the SDK is public channels, but users can specify "public_channel", "private_channel", "mpim", and "im"
+    # Multiple values can be passed for this argument as a comma separated list
+    # By default archived channels are NOT included by the SDK. Explicitly set this if not set from the CLI or set to False
+    body = {
+        'types': args.get('channel_types'),
+        'exclude_archived': argToBoolean(args.get('exclude_archived', 'true')),
+        'limit': args.get('limit')
+    }
+    if args.get('cursor'):
+        body['cursor'] = args.get('cursor')
+    raw_response = send_slack_request_sync(CLIENT, 'conversations.list', http_verb="GET", body=body)
+    # Provide an option to only select a channel by a name. Instead of returning a full list of results this allows granularity
+    # Supports a single channel name
+    name_filter = args.get('name_filter')
+    if name_filter:
+        channels = None
+        for channel in raw_response['channels']:
+            if channel['name'] == name_filter:
+                channels = [channel]
+                break
+        if channels is None:
+            return_error(f"No channel found with name: {name_filter}")
+    else:
+        channels = raw_response['channels']
+    # Force list for consistent parsing
+    if isinstance(channels, dict):
+        channels = [channels]
+    context = []  # type: List
+    if isinstance(channels, list):
```

Comment:
why wound't it be? 🤔 
I mean, we just wrapped a dictionary with `[]` three lines up